src/core/serializedtypes.js

```diff
@@ -308,25 +296,25 @@ const STInt64 = exports.Int64 = new SerializedType({
       if (value < 0) {
         throw new Error('Negative value for unsigned Int64 is invalid.');
       }
-      bigNumObject = new SJCL_BN(value, 10);
+      bigNumObject = new BN(value, 10);
     } else if (isString(value)) {
       if (!isHexInt64String(value)) {
         throw new Error('Not a valid hex Int64.');
       }
-      bigNumObject = new SJCL_BN(value, 16);
-    } else if (value instanceof SJCL_BN) {
-      if (!value.greaterEquals(0)) {
+      bigNumObject = new BN(value, 16);
+    } else if (value instanceof BN) {
+      if (value.ucmp(new BN(0)) === -1) {
         throw new Error('Negative value for unsigned Int64 is invalid.');
       }
       bigNumObject = value;
     } else {
-      throw new Error('Invalid type for Int64');
+      throw new Error('Invalid type for Int64: ' + (typeof value) + ' value');
     }
-    serializeBits(so, bigNumObject.toBits(64), true); // noLength = true
+    serializeBytes(so, bigNumObject.toArray('be', 8), true); // noLength = true
```

Comment:
'be' means big endian, and the following arg is the byte length, which it will pad with 0s if not enough bytes, or throw if over